draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -848,6 +848,152 @@ Content-Type: text/plain
 </artwork>
 </section>
 
+<section title="Origin" anchor="http.origin">
+<t>
+   The origin for a given URI is the triple of scheme, host, and port
+   after normalizing the scheme and host to lowercase and normalizing the
+   port to remove any leading zeros. If port is elided from the URI, the
+   default port for that scheme is used. For example, the URI
+</t>
+<sourcecode>
+   https://Example.Com/happy.js
+</sourcecode>
+<t>
+   would have the origin
+</t>
+<sourcecode>
+   { "https", "example.com", "443"}
+</sourcecode>
+<t>
+   which can also be described as the normalized URI prefix with port always
+   present:
+</t>
+<sourcecode>
+   https://example.com:443
+</sourcecode>
+<t>
+   The notion of an origin for any given URI is central to the semantics of
+   HTTP. Each origin defines its own namespace and controls how identifiers
+   within that namespace are mapped to resources. In turn, how the origin
+   responds to valid requests, consistently over time, determines the
+   semantics that users will associate with a URI, and the usefulness of
+   those semantics is what ultimately transforms these mechanisms into a
+   "resource" for users to reference and access in the future.
+</t>
+<t>
+   The general term "authority" is also used as an abstract reference to
+   whatever entity controls a given resource. A single authority might control
+   only parts of an origin, an entire origin, or many different origins.
```

Comment:
Though a single logical authority might control a subset of the resources attributed to an origin, or even resources that span multiple origins, I don't know that this is a helpful statement to make here.  You can recognize that "authority" is a concept that HTTP uses both in this more abstract sense, but the protocol only concretely cares about authority in a very specific sense.  That is, the concrete definition of authority in HTTP is based on the origin of the resource alone.

Reply:
Yep, thanks for the careful review. I can delete the paragraph and just use entity below.